lib/components/user/monitored-trip/saved-trip-screen.js

```diff
@@ -191,27 +205,37 @@ class SavedTripScreen extends Component {
             // and to its own blur/change/submit event handlers that automate the state.
             // We pass the Formik props below to the components rendered so that individual controls
             // can be wired to be managed by Formik.
-            (props) => {
-              return (
-                <Form noValidate>
-                  <SavedTripEditor
-                    {...props}
-                    isCreating={isCreating}
-                    notificationChannel={loggedInUser.notificationChannel}
-                    onCancel={
-                      isCreating ? this._goToTripPlanner : this._goToSavedTrips
-                    }
-                    panes={this._panes}
-                  />
-                </Form>
-              )
-            }
+            (props) => (
+              <Form noValidate>
+                <SavedTripEditor
+                  {...props}
+                  isCreating={isCreating}
+                  notificationChannel={loggedInUser.notificationChannel}
+                  onCancel={
+                    isCreating ? this._goToTripPlanner : this._goToSavedTrips
+                  }
+                  panes={this._panes}
+                />
+              </Form>
+            )
           }
         </Formik>
       )
     }
 
-    return <AccountPage>{screenContents}</AccountPage>
+    return (
+      <AccountPage>
+        <InvisibleA11yLabel role="status">
```

Comment:
Curious what your thoughts are on combining this status bar with the other existing status bar on the page? I tested with safari + voiceover and this does seem to be working okay, but I am wondering if having multiple status regions could be confusing or play differently on different AT

Reply:
For "polite" live regions, it seems fine [per this post as long as the updates aren't caused by the same action](https://gaurav5430.medium.com/quick-accessibility-wins-multiple-aria-live-on-single-action-caveat-b79a6f41e7cc).